pytests/test_compatibility.py

```diff
@@ -10,16 +10,18 @@
 
 
 class TempDirectory(object):
-    def __init__(self):
+    def __init__(self, suffix=""):
```

Comment:
This change feels wrong. Temp dir should be ... temporal, so the name does not matter. If you want to distinguish between test in debug - create root temp directory with named directories inside. While this ad-hoc debug technique might be helpful, I think it should not be merged.

Reply:
I changed it such that one can decide whether to have temporary files in the system's temp dir (i.e. the old behavior) or to keep the results in a separate folder (for manual inspection after the tests).

Reply:
It resides in tests only, so let it be.

Reply:
Just a thought. We have TempDir guard in main extension code too. I think switching temp dir to known place might be helpful feature in debugging on user's side, note #125.